index.html

```diff
@@ -162,7 +162,7 @@ <h2 class="text-center">
                     </a>
                 </h2>
                 <div class="underline text-center"></div>
-            </div>
+            </div> 
         </section>
```

Comment:
- [ ] OPTIONAL: I admire your creativity in your design but design requirements are always business/client requirements and you are not supposed to change it. There are differences between the original and your design in some sections. Background color, contrast, typography, spacing, alignment, positioning of the different elements should be strictly similar to the original design. Even though it is not a mandatory requirement because it was not mentioned by the previous reviewer I strongly suggest you to make your design resembles very closely to the original design. You personalize the contents not the design. :pray: 

![image](https://user-images.githubusercontent.com/64894927/132727386-c9598bea-698e-4667-a2f2-0221b143b66d.png)

![image](https://user-images.githubusercontent.com/64894927/132727476-c72af8a3-ee12-4138-ba1d-4d4bf5f7f013.png)

In the about page 
![image](https://user-images.githubusercontent.com/64894927/132729201-38554419-438a-4364-8ed3-1af143ed1aef.png)
![image](https://user-images.githubusercontent.com/64894927/132729270-19171eb6-404e-4666-9516-81d40aba9050.png)